projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -364,6 +364,11 @@ function_exists( 'wpcom_site_has_feature' ) &&
 		// would conflict with our own Settings > Performance that links to Calypso, so we hide it it since the Calypso
 		// performance settings already have a link to Page Optimize settings page.
 		$this->hide_submenu_page( 'options-general.php', 'page-optimize' );
+
+		// Add Avertising menu item.
+		if ( apply_filters( 'dsp_promote_posts_enabled', false, get_current_user_id() ) ) {
```

Comment:
Could you add a docblock for this new filter, so it can be picked up by the parser and documented in [the Jetpack codex](https://developer.jetpack.com/) with the other hooks?

You can add it here since you're editing this file, and then add `/** This filter is documented in  projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php */` in the other place where the filter was introduced without a docblock last week (see #25726).

I would also recommend adding a `jetpack_` prefix to the filter name, to remain consistent with other filters in use in the plugin.

For the `@since` entry of the docblock, you can specify `@since $$next-version$$`. This way you do not have to care about the exact version number; it will be added for you on the next plugin release.
For reference:
https://github.com/Automattic/jetpack/blob/3a3c4adc1c0913bac224ddb832713bd6101fe1ab/projects/packages/README.md#L109-L116

Reply:
I've added your suggestions on the atomic-admin-menu class. The change on https://github.com/Automattic/jetpack/pull/25726 will be done at a later stage since it will need changes on the WPCOM side on a piece of code that's currently being heavily used in a CfT, context: pdtkmj-pb-p2, pMyNb-5CW-p2